Sources/AppStoreConnectClient/AppStoreConnectClient.swift

```diff
@@ -124,20 +124,26 @@ public struct AppStoreConnectClient {
     /// - Throws: An error of type `AppStoreConnectError` if the response indicates an error.
     public func fetchBuilds(
         for app: Application,
-        with query: BuildsQuery
+        with query: BuildsQuery? = nil
     ) async throws -> [Build] {
-        let response = try await client.builds_hyphen_get_collection(
-            query: .init(
-                filter_lbrack_app_rbrack_: [app.id],
-                sort: type(of: query.sort).init(arrayLiteral: ._hyphen_version),
-                fields_lbrack_builds_rbrack_: type(of: query.fields).init(
-                    arrayLiteral: 
-                    .version,
-                    .minOsVersion,
-                    .uploadedDate
+        let response: Operations.builds_hyphen_get_collection.Output
```

Comment:
I'm not sure that this is correct, please double check

Reply:
Checked.